TundraTD/Assets/Scripts/City/Building/Upgrades/IUpgrade.cs

```diff
@@ -0,0 +1,19 @@
+using Spells;
+using UnityEngine;
+
+namespace City.Building.Upgrades
+{
+    /// <summary>
+    /// Describes the behaviour of any Upgrade in a game
+    /// </summary>
+    public interface IUpgrade
+    {
+        Sprite Sprite { get; }
```

Comment:
I think it would be better to comment interface properties because there are many classes that implement this interface